mender-update/deployments/deployments.cpp

```diff
@@ -285,7 +285,7 @@ error::Error DeploymentClient::PushStatus(
 			auto resp = exp_resp.value();
 			auto content_length = resp->GetHeader("Content-Length");
 			if (!content_length) {
-				log::Error(
+				log::Debug(
```

Comment:
Right, I missed that this can happen (actually, it can *only* happen) when there is no body. But that is indeed possible here.

However, this means that the rest of the code is wrong: Since it can only happen when there is no body, we should not set `body_writer->SetUnlimited(true)` (it won't misbehave, but it masks the intention), so I would remove that in addition to the fix you did.